src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-29 calendar--start-wed">
```

Comment:
```suggestion
    <div class="calendar calendar--month-length--29 calendar--start-day--wed">
```
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/be510e33-2233-4380-898f-1c353b5c7e10)
Not critical but pay attention to the task description